packages/core/src/particle/ParticleRenderer.ts

```diff
@@ -590,8 +588,8 @@ export class ParticleRenderer extends GeometryRenderer {
     return material;
   }
 
-  private _createGeometry(): BufferGeometry {
-    const geometry = new BufferGeometry(this._entity.engine, "particleGeometry");
+  private _createGeometry(): Mesh {
```

Comment:
Rename _createGeometry to _createMesh should be better?

Reply:
Yes！